index.html

```diff
@@ -7,45 +7,39 @@
 <link rel="stylesheet" type="text/css" href="styles.css">
 </head>
 <body >
-    <div class="black__interface">
-    <div class="Autherr__Background"> 
+    <div class="Auther__Information">
           <div class="Auther__Content">
-             <div class ="Auther__BackgroundImg" >
-                <img  src="9.png" alt="Background" width="600" height="250" >
-                <img class="Auther__BackgroundImg--Sticker1" src="12.png" alt="Background" width="20" height="20" >
-                <img class="Auther__BackgroundImg--Stickerpower" src="3.png" alt="Background" width="30" height="20" >
+             <div>
+                <img class ="Auther__BackgroundImg" src="9.png" alt="Background"  >
+                <img class="Auther__Sticker1" src="12.png" alt="Background" width="5%" height="5%" >
+                <img class="Auther__StickerPower" src="3.png" alt="Background" width="5%" height="5%" >
+                <img class="Auther__StickerWaves" src="4.png" alt="Background" width="5%" height="5%" >
+                <img class="Auther__Img" src="john.jpg" alt="john smith" >

```

Comment:
the classes here should be renamed to Content__BackgroundImg, intead od Author, as they are nested inside the Content